.yamllint.yml

```diff
@@ -38,3 +36,4 @@ rules:
   trailing-spaces: enable
   truthy:
     level: error
+    check-keys: false
```

Comment:
Is there a reason why we don't check keys?

Reply:
Because I personally don't like it when yaml keys are in quotes:

See for example: https://github.com/adrienverge/yamllint/issues/430

If `check-keys` is set to `true`, yamllint will fail on:

```sh
yes:  1
on:   2
True: 3
```

That said, my opinion should not be leading here. Let's discuss

Reply:
I wonder when we will have yes/on/True or other truthy values as keys.
So my thought was we should warn when that accidentally happens.

But I agree, quoted keys are not ideal. But I hope this will never be required.

Reply:
We already use it in our GH workflows:

`on: ["push", "pull_request"]`

Hence I propose we keep check-keys disabled.